### PR TITLE
Compat RF - Improve/Change RF Realistic Name

### PR DESCRIPTION
**Pull request by Apricot-ale**
**When merged this pull request will:**
- _Add HEMMT Fire Truck localization (for Equalization with A3 realistic name)_
- _Change Puma family naming and add missing variation_
- _Change HADES(Hera H6) name to Uppercase_

### IMPORTANT

- If the contribution affects [the documentation](https://github.com/acemod/ACE3/tree/master/docs), please include your changes in this pull request so the documentation will appear on the [website](https://ace3.acemod.org/).
- [Development Guidelines](https://ace3.acemod.org/wiki/development/) are read, understood and applied.
- Title of this PR uses our standard template `Component - Add|Fix|Improve|Change|Make|Remove {changes}`.

### CODE REVIEW

**Comment by Apricot-ale** [comment]
A I would like to hear your opinion on this. Regarding the RAI-350/360, which is thought to be based on the IAR-330, I would like you to consider whether it should remain the Puma, which has a similar shape, or whether it should be made into the IAR-330, which it is likely to be based on.

File: addons/compat_rf/compat_rf_realisticnames/CfgVehicles.hpp [hunk 1]
```diff
@@ -19,36 +19,54 @@ class CfgVehicles {
         displayName = SUBCSTRING(heli_light_03_unarmed_Name);
     };
 
-    class Heli_EC_01A_base_RF;
-    class Heli_EC_01A_military_base_RF: Heli_EC_01A_base_RF {
-        displayName = SUBCSTRING(ec_01a_military_Name);
-    };
-
+    //H240 Transport, Gendarmerie/ION Transport
```
Comment (johnb432) [inline addons/compat_rf/compat_rf_realisticnames/CfgVehicles.hpp:22]: Add spaces between the start of the comment and the first word, that way it's uniform with the rest of the file.
```suggestion
    // H240 Transport, Gendarmerie/ION Transport
```

Comment (Apricot-ale) [reply addons/compat_rf/compat_rf_realisticnames/CfgVehicles.hpp]: Added!

**Review by johnb432 — COMMENTED** [review]
I don't have RF or the compat mod, so I can't weigh in properly, but I'm happy with the formatting. Name change from `Hera` to `HERA` also makes sense to me.

**Comment by Apricot-ale** [comment]
> I don't have RF or the compat mod, so I can't weigh in properly, but I'm happy with the formatting. Name change from to also makes sense to me.`Hera``HERA`

Thanks, and I'll one more change to this (AW159 thing)

**Comment by Apricot-ale** [comment]
I think it's ok

**Review by PabstMirror — APPROVED** [review]

**Pull request merged**